### PR TITLE
Reference tests for patterns starting with special chars

### PR DESCRIPTION
**Pull request by stasm**
A few tests for https://github.com/projectfluent/fluent.js/issues/512.

### CODE REVIEW

**Comment by stasm** [comment]
Ah, good point, I went by the test case from the original issue, but going by the EBNF, the test cases can be made more minimal.

**Comment by stasm** [comment]
I think Travis is having a hiccup but I'll look again into the CI error tomorrow.

File: test/fixtures/special_chars.ftl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+bracket-inline = [Value]
```
Comment (Pike) [inline test/fixtures/special_chars.ftl:1]: You're missing the 

```properties
## OK

```

Comment (stasm) [reply test/fixtures/special_chars.ftl:1]: Grr, sorry, my bad. I removed it to re-trigger the build and forgot to re-add it :/

**Pull request merged**